core/src/main/java/org/dozer/el/NoopELEngine.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2005-2017 Dozer Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dozer.el;
+
+import java.lang.reflect.Method;
+
+public class NoopELEngine implements ELEngine {

```

Comment:
I would appreciate javadoc here, so that users could easily see what will be the default behaviour of their bean mapper.

Reply:
Updated as requested.